coderd/workspaces.go

```diff
@@ -1848,6 +1849,144 @@ func (api *API) watchWorkspace(rw http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// @Summary Watch workspace by ID via WebSockets
+// @ID watch-workspace-by-id-via-websockets
+// @Security CoderSessionToken
+// @Produce json
+// @Tags Workspaces
+// @Param workspace path string true "Workspace ID" format(uuid)
+// @Success 200 {object} codersdk.ServerSentEvent
+// @Router /workspaces/{workspace}/watch-ws [get]
+func (api *API) watchWorkspaceWs(rw http.ResponseWriter, r *http.Request) {
```

Comment:
I think we can try to avoid copy-pasting in HTTP handlers, using smth like this:
https://github.com/coder/coder/blob/yevhenii/one-way-ws-1-pr-review/coderd/workspaces.go#L1722-L1858

Reply:
I think we can drop SSE entirely. There isn't much value in maintaining 2 approaches imo.

Reply:
@evgeniy-scherbina I forgot the cli uses sse. You are right, we need to keep it for some time :cry: